lib/mws/products/client.rb

```diff
@@ -7,7 +7,8 @@ module Products
     # sourcing and pricing decisions for listing those products on Amazon
     # Marketplace websites.
     class Client < ::Peddler::Client
-      path '/Products/2011-10-01'
+      version '2011-10-01'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.